Products/DataCollector/plugins/zenoss/cmd/linux/netstat_rn.py

```diff
@@ -98,8 +98,8 @@ def parse(self, results):
             if isroute:
                 # 192.168.99.0/24 dev eth0  scope link
                 route.id = isroute.group(1).replace('/', '_')
-                route.mask = isroute.group(1).split('/', 1)[1]
-                if route.mask == 32:
+                route.routemask = isroute.group(1).split('/', 1)[1]
+                if route.routemask == 32:
```

Comment:
It would appear that route.routemask could never == 32 here since it was just parsed out of a string without conversion to int. IpRouteEntry.routemask is defined as type string. So probably best to keep working with it as a string.